crates/transaction-pool/src/pool/blob.rs

```diff
@@ -136,10 +138,20 @@ impl<T: PoolTransaction> BlobTransactions<T> {
         transactions
     }
 
+    /// Resorts the transactions in the pool based on the pool's current [PendingFees].
+    pub(crate) fn reprioritize(&mut self) {
+        for tx in self.by_id.values_mut() {
+            tx.update_priority(&self.pending_fees);
+        }
+    }
```

Comment:
the issue here is that this only updates the `BlobOrd` in

```rust
struct BlobTransaction<T: PoolTransaction> {
    /// Actual blob transaction.
    transaction: Arc<ValidPoolTransaction<T>>,
    /// The value that determines the order of this transaction.
    ord: BlobOrd,
}
```

the sorted list 

```rust
    /// _All_ transactions sorted by blob priority.
    all: BTreeSet<BlobTransaction<T>>,
```

is unaffected by this

do we need the BlobOrd as a value? in `by_id`?
if not we should just drain the `all` map, update the priority and recollect

Reply:
we need them to be consistent, but this is a good point, changed this to drain, update, and recollect. `by_id` is also updated now